todo_app/flask_config.py

```diff
@@ -6,5 +6,10 @@ def __init__(self):
         """Base configuration variables."""
         self.SECRET_KEY = os.environ.get("SECRET_KEY")
         self.LOGIN_DISABLED = os.getenv("LOGIN_DISABLED") == "True"
+        if os.getenv("LOG_LEVEL"):
+            self.LOG_LEVEL = os.getenv("LOG_LEVEL")
+        else:
+            self.LOG_LEVEL = "DEBUG"
```

Comment:
The "getenv" method (like the dictionary ".get" method) lets us specify a default value to shortcut this a bit:
```suggestion
        self.LOG_LEVEL = os.getenv("LOG_LEVEL", "DEBUG")
```